tests/attr/test_deconvolution.py

```diff
@@ -68,7 +71,12 @@ def test_deconv_matching(self):
         inp = 100.0 * torch.randn(1, 1, 4, 4)
         self._deconv_matching_assert(net, net.relu2, inp)
 
-    def _deconv_test_assert(self, model, test_input, expected, additional_input=None):
+    def _deconv_test_assert(
+            self,
+            model: Module,
+            test_input: TensorOrTupleOfTensors,
+            expected: List,
```

Comment:
Can you make the expected hint more specific, e.g List[List[float]]? It will likely be a Union including Tuples. Same for the functions below.

Reply:
done